ci/lib/stage-test-examples.jenkinsfile

```diff
@@ -222,23 +222,35 @@ stage('test-examples') {
             } catch (Exception e) { }
         }
     }
-
-    if (["ubuntu", "debian"].contains(env.os_release_id)) {
-       if (env.no_cpu.toInteger() > 16) {
-            try {
-                timeout(time:30,unit: 'MINUTES') {
-                  sh '''
-                        cd CI-Examples/mysql
+    
+    if (env.no_cpu.toInteger() > 16) {
+        try {
+               timeout(time:30,unit: 'MINUTES') {
+               sh '''
+                  cd CI-Examples/mysql
+                '''
+                if (["ubuntu"].contains(env.os_release_id)) {
```

Comment:
For CentOS/RHEL, you can add environment variable SQL_LIBDIR in config-centos8.jenkinsfile.

If possible, simplify below logic

Reply:
There are few more optimizations that we have in mind but will need some time for exploration, so will take up as a separate story at a later time.